src/declarations/stencil-private.ts

```diff
@@ -501,21 +502,6 @@ export interface CollectionDependencyManifest {
   tags: string[];
 }
 
-/** OLD WAY */
-export interface Collection {
-  collectionName?: string;
-  moduleDir?: string;
-  moduleFiles?: any[];
-  global?: any;
-  compiler?: CollectionCompiler;
-  isInitialized?: boolean;
-  hasExports?: boolean;
-  dependencies?: string[];
-  bundles?: {
-    components: string[];
-  }[];
-}
```

Comment:
I don't believe this interface is necessary

Reply:
Yeah did a quick check and you replaces the few places this was referenced 👍